File: app/src/main/java/bj/rxjavaexperimentation/model/artistrelease/RootArtistReleaseResponse.java

package bj.rxjavaexperimentation.model.artistrelease;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

import bj.rxjavaexperimentation.model.common.Pagination;

public class RootArtistReleaseResponse
{
    @SerializedName("pagination")
    @Expose
    private Pagination pagination;
    @SerializedName("releases")
    @Expose
    private List<ArtistRelease> artistReleases = null;

    public Pagination getPagination()
    {
        return pagination;
    }

    public void setPagination(Pagination pagination)
    {
        this.pagination = pagination;
    }

    public List<ArtistRelease> getArtistReleases()
    {
        return artistReleases;
    }

    public void setArtistReleases(List<ArtistRelease> artistReleases)
    {
        this.artistReleases = artistReleases;
    }

}

File: app/src/main/java/bj/discogsbrowser/model/artistrelease/RootArtistReleaseResponse.kt
package bj.discogsbrowser.model.artistrelease

import bj.discogsbrowser.model.common.Pagination
import com.google.gson.annotations.SerializedName

/**
 * Created by Josh Laird on 19/05/2017.
 */
data class RootArtistReleaseResponse(val pagination: Pagination,
                                     @SerializedName("releases") val artistReleases: List<ArtistRelease>)